src/pages/404.tsx

```diff
@@ -0,0 +1,101 @@
+import type { NextPage } from "next";
+import { useRouter } from "next/router";
+import Navbar from "../components/section/Navbar";
+import Footer from "../components/section/Footer";
+
+const NotFoundPage: NextPage = () => {
+  const router = useRouter();
+
+  const handleGoHome = () => {
+    router.push("/").catch((error) => {
+      console.error("Navigation error:", error);
+    });
+  };
+
+  return (
+    <div className="relative flex min-h-screen flex-col bg-gray-100">
+      <Navbar />
+      <div className="relative flex flex-1 flex-col items-center justify-center px-4 text-center">
+        {/* 404 Text with Gradient */}
+        <div
+          className="absolute text-center font-bold"
+          style={{
+            fontFamily: "Inter",
+            fontSize: "720px",
+            fontWeight: 900,
+            lineHeight: "100px",
```

Comment:
Why are we using style tag if we have tailwind, are these not achievable with what tailwind offers?